src/gui/addnewtorrentdialog.ui

```diff
@@ -192,16 +192,6 @@
         </property>
        </spacer>
       </item>
-      <item row="0" column="0">
-       <widget class="QCheckBox" name="startTorrentCheckBox_2">
-        <property name="text">
-         <string>Start torrent</string>
```

Comment:
It was for a redundant "Start torrent" checkbox. It seemed to be the result of a copy/paste mistake someone made. It's not referenced anywhere in the code, and it obscured the real "Start torrent" checkbox, causing bugs.